aries_cloudagent/storage/tests/test_askar_storage.py

```diff
@@ -349,3 +351,22 @@ async def test_postgres_wallet_storage_works(self):
 
         await postgres_wallet.close()
         await postgres_wallet.remove()
+
+class TestAskarStorageSearchSession(AsyncTestCase):
+    @pytest.mark.asyncio
+    async def test_askar_storage_search_session(self):
+
+      with async_mock.patch(
+      "aries_cloudagent.storage.askar.AskarProfile"
+      ) as AskarProfile:
+        askar_profile = AskarProfile(None, True)
+        askar_profile_scan = async_mock.MagicMock()
+        askar_profile.store.scan.return_value = askar_profile_scan
+        askar_profile.settings.get.return_value = "walletId"
+
+        storageSearchSession = test_module.AskarStorageSearchSession(askar_profile, "filter", None)
+        await storageSearchSession._open()
+
+        assert storageSearchSession._scan == askar_profile_scan
+        askar_profile.settings.get.assert_called_once_with("wallet.id")
+        askar_profile.store.scan.assert_called_once_with("filter", None, profile="walletId")
```

Comment:
It would be better to test it with a tag_query than none. Please add it.

Reply:
Fixed